ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -279,56 +263,19 @@ class AlertsView extends React.Component {
           )}
           <AlertsViewControls
             dropdownOptions={id ? [] : alertDropdowns}
+            pageNums={pageNums}
             alertSummaries={alertSummaries}
             issueTrackers={issueTrackers}
             optionCollectionMap={optionCollectionMap}
-            fetchAlertSummaries={id => this.fetchAlertSummaries(id, true)}
+            fetchAlertSummaries={(id, update, page) =>
+              this.fetchAlertSummaries(id, update, page)
+            }
             updateViewState={state => this.setState(state)}
             bugTemplate={bugTemplate}
             user={user}
+            validated={validated}
```

Comment:
You don't need to pass `validated` in. It will be passed in via `{...this.props}`.